docs/dev/developer-guides/bridging/bridging-asset.md

```diff
@@ -173,7 +171,16 @@ Users must call the `withdraw` method on the L2 bridge contract, which will trig
 - After the method is called, the funds are unlocked from the L1 bridge and sent to the withdrawal recipient.
 
 ::: warning
-
 On the testnet environment, we automatically finalize all withdrawals, i.e., for every withdrawal, we will take care of it by making an L1 transaction that proves the inclusion for each message.
-
 :::
+
+## Custom bridges on L1 and L2
+
+To build a custom bridge, create a regular Solidity contract which extends the correct interface for the layer.
+
+- L1: [IL1Bridge.sol](https://github.com/matter-labs/era-contracts/blob/main/ethereum/contracts/bridge/interfaces/IL1Bridge.sol)
+- L2: [L2ERC20Bridge.sol](https://github.com/matter-labs/era-contracts/blob/main/zksync/contracts/bridge/L2ERC20Bridge.sol)
+
+The interfaces provide access to the zkSync Era SDK deposit and withdraw implementations.
+
+For more info, check out an example [custom bridge implementation](https://github.com/matter-labs/zksync-2-dev/blob/main/core/tests/ts-integration/tests/custom-erc20-bridge.test.ts).
```

Comment:
1. It's the private repo
2. It's just a test of how we did deploy it

Probably the best way is to point to our implementation of the bridges. 
https://github.com/matter-labs/era-contracts/blob/main/ethereum/contracts/bridge/L1ERC20Bridge.sol


https://github.com/matter-labs/era-contracts/blob/main/zksync/contracts/bridge/L2ERC20Bridge.sol